build.gradle

```diff
@@ -1,4 +1,10 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.4.0'
+        classpath 'com.google.gms:google-services:4.3.10'
```

Comment:
Please import newest version. Android Studio should inform you about that:
![image](https://user-images.githubusercontent.com/3758285/227258412-c1a81bed-a6b2-4daa-b62a-b3878d8c9a7e.png)